bower.json

```diff
@@ -1,36 +0,0 @@
-{
```

Comment:
Is removing this file necessary to migrate to yarn 2?

Reply:
@kevinbarabash No, I had to update the Yarn version in Netlify, where I realized it installs Bower and runs `bower install`. I've separated changes to #2372 and it's merged.